TodoApp/TodoApp.Api.Tests/ItemsControllerTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
+using NUnit.Framework;
+using TodoApp.Api.Controllers;
+using TodoApp.Api.Models;
+using TodoApp.Api.Tests.Utils;
+
+namespace TodoApp.Api.Tests
+{
+    [TestFixture]
+    public class ItemsControllerTests
```

Comment:
Way to complicated tests. You are referencing the Items inside the controller, which will be deleted in the future tasks. 
Your tests now works only as a check that you controller returns specified data, and will always work as you reference the specified data directly.